### PR TITLE
CLOUD-3421 - 7.2.4 install module is missing cekit version field

### PR DESCRIPTION
**Pull request by luck3y**
https://issues.jboss.org/browse/CLOUD-3421
Signed-off-by: Ken Wills <kwills@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for other issues
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Review by jmesnil — APPROVED** [review]

File: modules/7.2.4/module.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 schema_version: 1
 
 name: "eap-7.2.4"
+version: "1.0"
```
Comment (rgdoliveira) [inline modules/7.2.4/module.yaml:4]: shouldn't this change be in file modules/7.2.5/module.yaml ?

Comment (luck3y) [reply modules/7.2.4/module.yaml:4]: It should be in both of them, this PR was created before the 7.2.5 module PR was created.

Comment (dbkreling) [reply modules/7.2.4/module.yaml:4]: I created a separate PR to merge this to 7.2.5/module.yaml [0] and will merge both.

[0] https://github.com/jboss-container-images/jboss-eap-7-image/pull/166

**Review by dbkreling — APPROVED** [review]
PR #166 adds the change to the file in the 7.2.5 version. This one looks good to me.

**Pull request merged**